unity/Assets/Editor/Build.cs

```diff
@@ -12,43 +12,55 @@ public class Build {
 #else
 		var buildTarget = BuildTarget.StandaloneOSXIntel64;
 #endif
-        build(GetBuildName(), GetAllScenePaths(), buildTarget);
+        build(GetBuildName(), buildTarget);
     }
 
     static string GetBuildName() {
         return Environment.GetEnvironmentVariable("UNITY_BUILD_NAME");
     }
 
     static void Linux64() {
-        build(GetBuildName(), GetAllScenePaths(), BuildTarget.StandaloneLinux64);
+        build(GetBuildName(), BuildTarget.StandaloneLinux64);
     }
 
     static void WebGL() {
-        build(GetBuildName(), GetSceneFromEnv().ToList(), BuildTarget.WebGL);
+        build(GetBuildName(), BuildTarget.WebGL);
     }
 
-    static void build(string buildName, List<string> scenes, BuildTarget target) {
+    static void build(string buildName, BuildTarget target) {
         var defines = GetDefineSymbolsFromEnv();
         if (defines != "") {
             var targetGroup = BuildPipeline.GetBuildTargetGroup(target);
             PlayerSettings.SetScriptingDefineSymbolsForGroup(targetGroup, GetDefineSymbolsFromEnv());
         }
+
+        List<string> scenes = GetScenes();
+        foreach (string scene in scenes) {
+            Debug.Log("Adding Scene " + scene);
+        }
         BuildPipeline.BuildPlayer(scenes.ToArray(), buildName, target, BuildOptions.StrictMode | BuildOptions.UncompressedAssetBundle);
```

Comment:
I've found that if you don't change any of the scene files (i.e. when you change only scripts) replacing
```
BuildOptions.StrictMode | BuildOptions.UncompressedAssetBundle
```
with
```
BuildOptions.BuildScriptsOnly | BuildOptions.StrictMode | BuildOptions.UncompressedAssetBundle
```
can result in some non-trivial speedups on sequential builds. Not a super fair request but can we add in an option to `local_build` to only build scripts like this? I.e. ``--build_scripts_only t`` or similar?

Reply:
Thats a good suggestion. I'll handle that as a separate PR.